corehq/apps/sms/api.py

```diff
@@ -169,6 +175,12 @@ def send_sms(domain, contact, phone_number, text, metadata=None, logged_subevent
     return queue_outgoing_sms(msg)
 
 
+def _backend_enabled_for_domain(backend, domain):
+    if backend.get_api_id() == 'TURN':
+        return domain in TURN_IO_BACKEND.get_enabled_domains()
+    return True
```

Comment:
The usual pattern for checking if a toggle is enabled for a domain is

`SOME_TOGGLE.enabled(domain)`

Is there a reason that wasn't done here? If not, there are several other places in this PR that should be changed to use this pattern as well.

nit: could combine this into a single expression.
```suggestion
    return backend.get_api_id() == 'TURN' and TURN_IO_BACKEND.enabled(domain)
```

Reply:
@millerdev No real reason other than the fact that I was not aware that was the preferred way - although I can see why, much cleaner! 

Regarding the combined conditions: I specifically do not want it combined, because the combined version will return `False` when backed api id is not 'TURN', but I want it to return `True` in that case.

Reply:
Oh, sorry for not reading more carefully. Combined could be
```py
    return backend.get_api_id() != 'TURN' or TURN_IO_BACKEND.enabled(domain)
```

Reply:
@millerdev Yeah this works! 
(I guess if I gave it at least 1 second of thought I'd have seen this)